src/python/foglamp/data_purge/config.json

```diff
@@ -1 +1 @@
-{"enabled": true, "lastID": 1998276, "age": "2 minutes", "retainUnsent": true, "wait": "2 days"}
\ No newline at end of file
+{"age": "5 second", "enabled": true, "wait": "10 seconds", "retainUnsent": false, "lastID": 629102}
```

Comment:
@MarkRiddoch - I'm changing the config params to work against @ashwinscale... During that time, I'll also ask him where @sstefano69 is storing the lastID he sends to the historian.